{{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% block head %}
+<style media="screen">
+    /* Example table styling taken from http://www.w3schools.com/css/css_table.asp */
+    .dummy-class {
+        border-collapse: collapse;
+        width: 100%;
+    }
+
+    .dummy-class th, td {
+        padding: 8px;
+        text-align: left;
+        border-bottom: 1px solid #ddd;
+    }
+</style>
+{% endblock %}
+
+{% block content %}
+    <h3>mapping1:</h3>
```

Comment:
Can you stick these into a two-col grid layout? That way folks who might not be familiar with bootstrap can see that feature in action.